.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I would prefer two additional settings:
- "arrowParens": "always" (much better readability)
- "printWidth": 100 (works well with github GUI side by side comparison)

My configuration for my own projects looks like this:
```json
{
	"arrowParens": "always",
	"bracketSpacing": true,
	"printWidth": 120,
	"semi": true,
	"singleQuote": true,
	"tabWidth": 4,
	"trailingComma": "none",
	"useTabs": true
}
```

Feel free to select some of these settings

Reply:
Yes, we'll be adding more elements in the next patch.